kore/src/Kore/Internal/MultiAnd.hs

```diff
@@ -73,8 +76,7 @@ A non-empty 'MultiAnd' would also have a nice symmetry between 'Top' and
 newtype MultiAnd child = MultiAnd { getMultiAnd :: [child] }
     deriving (Eq, Ord, Show)
     deriving (Semigroup, Monoid)
-    deriving (Functor, Applicative, Monad, Alternative)
```

Comment:
Oh, I had no idea this type defined an `Alternative` instance. That's really bad!